lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.upload_block_blob_from_file('<container name>', '<blob name>', '<file path>')
+puts "#{new_blob.inspect}"
+```
+
+## Download a blob to a local file
+```ruby
+blob = azure_storage_service.download_blob_to_file('<container name>', '<blob name>', '<file path>')
+puts "#{blob.inspect}"
+puts "File Size: #{File.size <file_path>}"
+```
+
+## Delete the storage blob
+
+Mark the specified blob for deletion. The blob is later deleted during garbage collection.
+
+```ruby
+blob = azure_storage_service.blobs(container_name: '<container name>').get('<blob name>')
```

Comment:
This confuses me. Where I referenced to is the storage account. [Get a single storage account](https://github.com/fog/fog-azure-rm/blob/master/lib/fog/azurerm/models/storage/storage_accounts.rb#L32) is calling with service.collection(arg 1).get(arg 2). It is in the storage.md, quoted as below. Which pattern should I follow?

## Retrieve a single Storage Account

Get a single record of Storage Account

```ruby
storage_acc = azure_storage_service
                .storage_accounts(resource_group: '<Resource Group name>')
                .get('<Storage Account name>')
puts "#{storage_acc.name}"
```